### PR TITLE
test(e2e): added sort test for lighthouse leaderboard total column and removal of unnecessary timeouts from individual command

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

 added sort test for lighthouse leaderboard total column and removal of unnecessary timeouts from individual command

# Explain the feature/fix

test(e2e): added sort test for lighthouse leaderboard total column and removal of unnecessary timeouts from individual command

## Does this PR introduce a breaking change

No

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?--
-Yes (it is not one feature, feature addition plus some code cleaning is included
- [x] Did tests pass? 
-  Yes , the test added is passing
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
-  Not applicable
- [x] Was this feature demoed and the design review approved? 
- Not applicable

### CODE REVIEW

**Review by riginoommen — DISMISSED** [review]

File: e2e/integration/feedbacktests.js [hunk 1]
```diff
@@ -3,19 +3,19 @@ context( 'Test feedback form', () => {
...
+        cy.visit( Cypress.env( 'QA_HOST' ) +'feedback');
+        cy.get( '#username' ).type( Cypress.env( 'USERNAME' ) );
         cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
-        cy.get( '#submit',{timeout:10000} ).click();
+        cy.get( '#submit').click();
     } );
 
     it( 'Report bug', () => {
         cy.get( '#feedback-popup', { timeout: 10000 } ).click().wait( 1000 );
         cy.get( `[data-feedback-type="bug"]`, { timeout: 10000 } ).click();
```
Comment (hybridx) [inline e2e/integration/feedbacktests.js:14]: Do we still need the timeout here?

**Review by hybridx — APPROVED** [review]

**Review by akhilmhdh — COMMENTED** [review]
Can we remove all other timeout

**Comment by akhilmhdh** [comment]
Is it required

File: e2e/integration/feedbacktests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/feedbacktests.js:14]: fixed @hybridx https://github.com/1-Platform/one-platform/pull/1484/commits/cca93f81599192904ea85ae0e4b94a12495eb660

**Review by riginoommen — DISMISSED** [review]

**Review by akhilmhdh — APPROVED** [review]
LGTM

**Review by riginoommen — APPROVED** [review]

File: e2e/integration/feedbacktests.js (hunk 1 above)
Comment (riginoommen) [reply e2e/integration/feedbacktests.js]: If this is fixed can you please review and merge

Comment (riginoommen) [reply e2e/integration/feedbacktests.js]: @hybridx

**Review by hybridx — APPROVED** [review]

**Pull request merged**